bucket/dos2unix.json

```diff
@@ -21,6 +21,16 @@
     ],
     "checkver": {
         "url": "https://waterlan.home.xs4all.nl/dos2unix.html",
-        "re": "Latest version: (?<version>[\\d.]+) \\([\\d\\-]+\\)"
+        "re": "Latest version: ([\\d.]+) \\([\\d\\-]+\\)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://waterlan.home.xs4all.nl/dos2unix/dos2unix-$version-win64.zip"
```

Comment:
Why not sourceforge? Current URLs are from sourceforge

Reply:
Understood, but the URLs are on https://waterlan.home.xs4all.nl/dos2unix.html, so let's consider these URLs are "authoritative". Also, Sourceforge returns 429 errors during many excavator runs, so minimizing hitting sourceforge will decease these 429s.

Reply:
Then run checkver to change current URLs